cmd/devicePinger.go

```diff
@@ -45,7 +45,7 @@ func MeasurePingStats(beat *client.DeviceHeartbeat, apiOrigin string, host strin
 		pinger.Timeout = HeartbeatInterval * time.Second
 		pinger.Run() // blocking until done
 		updateICMPPing(beat, pinger.Statistics())
-		log.Info("Updated device heartbeat with ICMP ping result")
+		log.V(1).Info("Updated device heartbeat with ICMP ping result")
```

Comment:
Just curious what is the difference between V(1) and just log?

Reply:
verbosity level. more info at https://github.com/go-logr/logr